### PR TITLE
fix: Detect flying chests correctly

### PR DESCRIPTION
**Pull request by ShadowCat117**
image

### CODE REVIEW

**Review by kristofbolyai — APPROVED** [review]
Looks good, but can you hold off on merging until the CI update is merged, so I can test it?

**Comment by ShadowCat117** [comment]
> Looks good, but can you hold off on merging until the CI update is merged, so I can test it?

Yeah no problem

**Pull request merged**
